bin/cluster-test.sh

```diff
@@ -53,5 +53,9 @@ echo "Chroma Server is running at port $CHROMA_SERVER_HOST"
 echo "Pulsar Broker is running at port $PULSAR_BROKER_URL"
 echo "Chroma Coordinator is running at port $CHROMA_COORDINATOR_HOST"
 
-echo testing: python -m pytest "$@"
-python -m pytest "$@"
+# echo testing: python -m pytest "$@"
+# python -m pytest "$@"
+
+export CHROMA_KUBERNETES_INTEGRATION=1
```

Comment:
Shouldn't the integration test run against a minikube cluster?

Reply:
I don't follow, this file starts a minikube cluster